gutenberg.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Gutenberg
  * Plugin URI: https://github.com/WordPress/gutenberg
  * Description: Printing since 1440. This is the development plugin for the new block editor in core.
- * Requires at least: 5.3
+ * Requires at least: 5.5
```

Comment:
Do you think we can remove this entirely from here? I mean we already have this value defined in readme.txt isn't that enough?

Reply:
I think we should keep them. See https://core.trac.wordpress.org/ticket/46938

Reply:
Could add `Tested up to` here too, fwiw

Reply:
Thanks for clarifying. I will make sure we update `@wordpress/create-block` and its templates to follow the same approach.